spec/namespaces/nodes.yaml

```
@@ -545,15 +530,21 @@ components:
       description: The type to sample.
       schema:
         $ref: '../schemas/nodes._common.yaml#/components/schemas/SampleType'
-    nodes.info::path.metric:
+    nodes.info::path.node_id_or_metric:
       in: path
-      name: metric
-      description: Limits the information returned to the specific metrics. Supports a comma-separated list, such as http,ingest.
+      name: node_id_or_metric
+      description: |
+        Limits the information returned to a list of node IDs or specific metrics.
+        Supports a comma-separated list, such as node1,node2 or http,ingest.
       required: true
       schema:
```

NodeIds is already an array, so need to lift up the anyOf
```suggestion
      schema:
        anyOf:
          - title: node_id
            $ref: '../schemas/_common.yaml#/components/schemas/NodeIds'
          - title: metric
            type: array
            items:
              $ref: '../schemas/nodes.info.yaml#/components/schemas/Metric'
```